crates/aranya-daemon/src/daemon.rs

```diff
@@ -188,7 +205,7 @@ impl Daemon {
             let (send_effects, recv_effects) = tokio::sync::mpsc::channel(256);
 
             let invalid_graphs = InvalidGraphs::default();
-            let state = QuicSyncState::new(psk_store.clone())?;
+            let state = QuicSyncState::new(psk_store.clone(), conn_map)?;
```

Comment:
We're calling the QUIC sync server `SyncServer` while the client is called `QuicSyncState`. I think we should call the client `QuicSyncClient` or `SyncClient` to avoid confusion.
```suggestion
            let state = SyncClient::new(psk_store.clone(), conn_map)?;
```